config-user.mk.acr

```diff
@@ -34,6 +34,10 @@ INSTALL_MAN=@INSTALL_MAN@
 INSTALL_LIB=@INSTALL_LIB@
 
 VERSION=@VERSION@
+R2_VERSION=@R2_VERSION@
```

Comment:
This is not working! After you run `./configure` it just remains `R2_VERSION=@R2_VERSION@` and the text doesn't get replaced with the actual value.